examples/pytorch/graphsaint/modules.py

```diff
@@ -0,0 +1,76 @@
+import torch.nn as nn
+import torch.nn.functional as F
+import torch as th
+import dgl.function as fn
+
+
+class GCNLayer(nn.Module):
+    def __init__(self, in_dim, out_dim, bias=False, order=1,
+                 activation=None, dropout=0, batch_norm=False):
```

Comment:
The original implementation has an `aggr` argument [here](https://github.com/GraphSAINT/GraphSAINT/blob/6b3465fffcf764b9646bf571e2b4dc5d6aa74f3c/graphsaint/pytorch_version/layers.py#L54), which allows aggregating the results for an order greater than 1. I think their model combining GraphSAGE and Mixhop.